### DISCUSSION TITLE
que repo tenemos que mandar para el backend

### DISCUSSION THREAD

**Discussion by JosafatJimenezB** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#573] [Q&A] [answered]
Hola, mi duda es que repositorio es el que tenemos que enviar para el formulario

**Comment by BattiestWinter** [comment]
Tampoco sé jaja, supongo que el último repositorio de la misión. En mi caso mandaré el link de la práctica de VUE

**Reply by jstoledano** [reply]
Es la dirección de tu perfil.

**Reply by JosafatJimenezB** [reply]
gracias

**Comment by carlos-gmx3** [comment, marked as answer]
La liga a tu perfil de Github, lo que checan es que seas parte de la organización de Launch-X
en mi caso fue:
https://github.com/carlos-gmx3

**Reply by chistoperez** [reply]
pero dice liga de tu repositorio, no de tu perfil,
no sería https://github.com/carlos-gmx3?tab=repositories 
?

**Reply by carlos-gmx3** [reply]
En palabras de Fer en el minuto 43 con 23 segundos
https://youtu.be/6zdpgm8kwQY?t=2604
![imagen](https://user-images.githubusercontent.com/99064239/161364973-a42b8ad8-dcbb-46f1-bde3-91a22cb43e4f.png)

**Reply by RodolfoMorquecho** [reply]
Alguien sabe hasta cuando tenemos para enviar todo lo requerido?

**Reply by jstoledano** [reply]
> Alguien sabe hasta cuando tenemos para enviar todo lo requerido? 

Nomas es llenar un formulario. No es la gran cosa.

**Reply by RodolfoMorquecho** [reply]
> > Alguien sabe hasta cuando tenemos para enviar todo lo requerido?
> 
> Nomas es llenar un formulario. No es la gran cosa.

Es que como mencionan aquí que se tiene que enviar el link del ultimo repositorio y aun no termino la practica de vuex por eso conteste así, perdón.

**Reply by jstoledano** [reply]
Yo creo que su sistema nomás verifica que exista la cadena "github.com/usuario"; no verifica realmente un repositorio en específico y la respuesta es inmediata.

**Reply by JosafatJimenezB** [reply]
gracias

**Reply by carlos-gmx3** [reply]
Checa el video de la transmisión, es el link a tu perfil. El script que hayan escrito para automatizar el proceso solo revisa que seas parte de la organización Launch-X
Tampoco yo he terminado lo de Vue, estoy tomando un curso para poder hacerlo bien, la pagina de pastelería que ando haciendo es de una amiga donde trabajaba en bazares vendiendo pasteles y pues hay posibilidad de que se vuelva un proyecto real.
Poco a poco como decía el MC de Misión Frontend

**Reply by FernandoBarajasValdivia** [reply]
Ya les llegó la invitación al repo de back?
![image](https://user-images.githubusercontent.com/99165285/161367130-8808b187-a628-41ef-bfc9-9712f60bfff9.png)

**Reply by FernandoBarajasValdivia** [reply]
A mi no :(

**Reply by JosafatJimenezB** [reply]
Tampoco me llego

**Reply by carlos-gmx3** [reply]
Tampoco x3
Solo revisa que ya tengas acceso al equipo
https://github.com/orgs/LaunchX-InnovaccionVirtual/teams
![image](https://user-images.githubusercontent.com/99064239/161405666-bc532cd5-c9be-4b2c-904e-e356566f3681.png)
Si lo puedes ver, ya estás dentro
![image](https://user-images.githubusercontent.com/99064239/161405678-4890b976-b942-42fd-a282-332a3be92bec.png)
Igual checa en Microsoft Teams si ya te agregaron el equipo
![image](https://user-images.githubusercontent.com/99064239/161405699-679e5a59-3bd7-4fb1-86a8-919b04ea9d2a.png)

**Reply by BattiestWinter** [reply]
> Ya les llegó la invitación al repo de back? ![image](https://user-images.githubusercontent.com/99165285/161367130-8808b187-a628-41ef-bfc9-9712f60bfff9.png)

Si te llegó la invitación? es que a mi me sigue sin llegar aunque introduzca mi usuario

**Reply by JosafatJimenezB** [reply]
checa si ya te aparece el repo por que no te llega ningun correo, solo te agregan

**Reply by BattiestWinter** [reply]
Aún no, sólo veo el repositorio de nodejs, pero porque es público. Supongo que debe de ser un problema general porque si es muy grande la diferencia entre miembros de nodejs y java

**Comment by DanyVeneno** [comment]
Yo pregunte en DM en Instagram y me dijeron que el ultimo yo puse la direccion de el ejercicio de Vue aunque todavia no lo termino
